### ISSUE TITLE
Menu crashes the app when ionic_experimental_manual_treeshaking=true.

### ISSUE THREAD

**Issue by kleeb** [driftyco/ionic-app-scripts#816]
Ionic 2.2.0
ionic-app-scripts 1.1.4

After building rollup release bundle with 

```
ionic_experimental_manual_treeshaking=true
ionic_experimental_purge_decorators=true
```

app crashes on the menu toggle (ion-menu) https://ionicframework.com/docs/v2/api/components/menu/Menu/

```
EXCEPTION: Uncaught (in promise): TypeError: menuTypes[e] is not a constructor
TypeError: menuTypes[e] is not a constructor
    at Function.e.create (http://localhost:8100/build/main.js:16:12527)
    at e._getType (http://localhost:8100/build/main.js:22:30092)
    at http://localhost:8100/build/main.js:22:30390
    at new t (http://localhost:8100/build/polyfills.js:3:11969)
    at e.setOpen (http://localhost:8100/build/main.js:22:30352)
    at e.toggle (http://localhost:8100/build/main.js:23:272)
    at e.toggle (http://localhost:8100/build/main.js:23:8235)
    at e.handleEvent (http://localhost:8100/build/main.js:83:8498)
    at t.handleEvent_11 (http://localhost:8100/build/main.js:98:10124)
    at HTMLButtonElement.<anonymous> (http://localhost:8100/build/main.js:2:1607)
    at t.invokeTask (http://localhost:8100/build/polyfills.js:3:10284)
    at Object.onInvokeTask (http://localhost:8100/build/main.js:7:30521)
    at t.invokeTask (http://localhost:8100/build/polyfills.js:3:10220)
    at e.runTask (http://localhost:8100/build/polyfills.js:3:7637)
    at HTMLButtonElement.invoke (http://localhost:8100/build/polyfills.js:3:11397)
```

Turning off the ionic_experimental_manual_treeshaking fixes this bug.

**Comment by numerized** [comment]
Hi guys, 

Same here with Webpack and ionic-app-scripts 1.3.1 and Ionic 3.0.1

**Comment by jgw96** [comment]
Hello all! With the latest release of app-scripts (1.3.7) do you still get the same error?

**Comment by kleeb** [comment]
Looks like its fixed with 1.3.7 and ionic 3.2.1

**Comment by numerized** [comment]
Fixed for me too
